docs/tutorials/exploring_bdk_rn.md

```diff
@@ -0,0 +1,704 @@
+    });
+    setTransaction(data);
+    setDisplayText(JSON.stringify(data));
+  };
+```
+
+ add a new section for send transaction functionality. We will need an input box for receiver address and an input box for amount to send. We will also need a button to trigger the transaction.
+
+Let's add the send transaction section and ui components below `{/* input boxes and send transaction button */}`
+
+```jsx
+    {/* input boxes and send transaction button */}
+    <View style={styles.sendSection}>
+      <Fragment>
+        <TextInput
+          style={styles.input}
+          placeholder="Recepient Address"
+          onChangeText={setRecepient}
+        />
+        <TextInput
+          style={styles.input}
+          placeholder="Amount (in sats)"
+          onChangeText={e => setAmount(parseInt(e))}
+        />
+        <Button
+          title="Send Transaction"
+          style={styles.methodButton}
+          onPress={sendTx}
+        />
+      </Fragment>
+    </View>
+```
+
+
+We should now be able to send a transaction as long as there is sufficient balance. 
+
+<img src="./exploring_bdk_rn/bdk_rn_send.png" style="display: block; margin: 0px auto; zoom: 50%;" />
+
+
+
+## Conclusion
+
+The app we created can be distributed for both iOS and Android thus sharing code base, the development focused on use cases and we did not have to code intricate bitcoin node, transaction and sync related functionalities this was managed by `bdk-rn` allowing the product to focus on the functionality. This is what `bdk` and `bdk-rn` intend to do, make Rapid Bitcoin Application Development possible.
+
+
+
+bdk-rn intends to expose functionality and api from bdk which has a wide variety of api with granular details allowing for many interesting use cases to be implemented. bdk-rn and bdk are constantly updated and enhanced based on developer feedback both from bitcoin community as well as bitcoin product developmnt teams.
+
+
+
+Stay tuned for more api's and enhancements coming to bdk-rn in the near future. Feature and api requests are most welcome. New blogs and tutorials will be published soon for more in depth exploration of bdk-rn. 
+In the meantime keep in touch with the project on [GitHub](https://github.com/LtbLightning/bdk-rn) and [Twitter](https://twitter.com/BitcoinZavior).
+
+
+#### References:
+
+- [bdk](https://github.com/bitcoindevkit)
+
+- [bdk-rn](https://github.com/LtbLightning/bdk-rn)
+
+- [Awesome Bitcoin App Github Repository](https://github.com/LtbLightning/AwesomeBitcoinApp)
+
+- [Setup React Native Development Environment](https://reactnative.dev/docs/environment-setup)
+
+- [Mastering Bitcoin(HD Wallet chapter)](https://www.oreilly.com/library/view/mastering-bitcoin/9781491902639/ch04.html)
+
+- [Bitcoin Output Descriptors from bitcoin GitHub](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md)
+
+- [Testnet Faucet]( https://coinfaucet.eu/en/btc-testnet/ or https://bitcoinfaucet.uo1.net/)
+
+  
+
+
+
+
```

Comment:
Last comment, you have some images that are more than 1MB. I think you should compress those. You can use this free service, https://tinypng.com/.  I will recommend you pass the article through Grammarly, it will help you catch a lot of the grammar issues.

Reply:
Done ✅